### ISSUE TITLE
the ".swifthlm" account must created by manual

### ISSUE THREAD

**Issue by JasonCoded** [ibm-research/swifthlm#11]
we get the error log :  
      account server: “HEAD  /hd4/95/.swifthlm” 404

the ".swifthlm" account must created by manual ?

**Comment by hseipp** [comment]
@JasonCoded  No, the .swifthlm account is automatically created with the first PUT operation to the SwiftHLM request queue.
